src/index.ts

```diff
@@ -1,5 +1,5 @@
 export * from './use-swr'
-export { default } from './use-swr'
+export { default as useSWR } from './use-swr'
```

Comment:
```suggestion
import { default as useSWR } from './use-swr'
```

Reply:
Want to also export `useSWR` as a non-default? Or just this fix?

Reply:
@aequasi just this fix I think 🙏